bar-trender-front-end/src/components/ListDiscount.js

```diff
@@ -14,14 +14,15 @@ import {
 
 const ListDiscount = (props) => {
   const { discounts } = props;
-  //console.log(discounts.discounts)
-  if (!discounts.discounts || discounts.discounts.length == undefined) return <p>No repos, sorry</p>;
+  
+  console.log(discounts.results);
+  if (!discounts.results || discounts.results.length == 0) return <p>No hay descuentos para este establecimiento</p>;
```

Comment:
Codacy has a fix for the issue: [Expected { after 'if' condition.](https://app.codacy.com/gh/BarTrender/BarTrenderFront/pullRequest?prid=7167715)
```suggestion
  if (!discounts.results || discounts.results.length == 0) {return <p>No hay descuentos para este establecimiento</p>;}
```